src/edit/RotateHandle.js

```diff
@@ -19,6 +19,21 @@ L.RotateHandle = L.EditHandle.extend({
 		overlay.editing._rotateBy(angle);
 		overlay.editing._scaleBy(scale);
 
+		/* 
+		  checks whether the "edgeMinWidth" property is set and tracks the minimum edge length;
+		  this enables preventing scaling to zero, but we might also add an overall scale limit
```

Comment:
I just added one more line of context here for the future... then this is good to merge! Thanks!

Reply:
Yes, i think we need to insert hard limit on minimum scale to prevent errors. I can do it